Makefile

```diff
@@ -3,7 +3,10 @@ all:
 
 deb: clean
 	@echo '*** See debian/README.source for more information and help with troubleshoting. ***'
-	dpkg-buildpackage -us -uc
+	@# We can't use --filename=jsdbg-gdb.deb in BUILDPACKAGE_OPTS because too many steps
+	@# depend on keeping the filename as-is. So we'll just rename it afterwards.
+	BUILDPACKAGE_OPTS="--destdir=." dpkg-buildpackage -us -uc --build=binary --buildinfo-option=-u. --changes-option=-u.
+	mv jsdbg-gdb_*.deb jsdbg-gdb.deb
```

Comment:
Should this be "mv **jsdbg_*.deb** jsdbg-gdb.deb"? The name of .deb package produced currently doesn't seem to include "gdb".

Reply:
Or is that what the package name change in the control file is about?

Reply:
The change to debian/control makes it produce jsdbg-gdb_0.9.0_amd64.deb

Reply:
Ok, makes sense.

Reply:
Technically no, but it is necessary to make jsdbg show up as jsdbg-gdb in the list of installed packages (e.g. dpkg -l), which I thought would be good to change if we rename the .deb file like that.

Reply:
Ok. Sounds good.